app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
```

Comment:
- [ ] Kindly fix the layout of the form elements to match the wireframe as the prev reviewer asked, make sure ensure it's responsive. It looks like this:
![image](https://user-images.githubusercontent.com/91486791/231754532-ab604df1-50d8-4c8a-8587-e7d384a38c27.png)
- I also noticed the edit user form page has the same problem. Kindly fix that too.